Create global constant for string literal 'blob_names'

enosair opened the pull request:
Reviewed By: boryiingsu

Differential Revision: D9484567

enosair commented:
@jerryzh168 could you review this? The failed building is irrelevant.

jerryzh168 commented:
@enosair you can land the diff in phabricator and this one will be merged. no need for stamp in GH.

Commit:
Create class constant for string literal 'blob_names'

Summary: Pull Request resolved: https://github.com/pytorch/pytorch/pull/10827

Reviewed By: boryiingsu

Differential Revision: D9484567

fbshipit-source-id: 9dd645b9a3141a42b83bc2350bacaf2c9830134b
